### PR TITLE
[FIX] update plot_3d_map_to_surface_projection

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none

Changes proposed in this pull request:

- slight adaptation of the example to reference the right functions
- create a surface image for the signed curvature

### CODE REVIEW

File: examples/01_plotting/plot_3d_map_to_surface_projection.py [hunk 1]
```diff
@@ -243,8 +250,8 @@
 # %%
 # Plot multiple views of the 3D volume on a surface
 # -------------------------------------------------
-#
-# :func:`~nilearn.plotting.plot_img_on_surf` takes a statistical map
+# :func:`~nilearn.plotting.plot_img_on_surf`
+# takes a niftistatistical map
```
Comment (Remi-Gau) [inline examples/01_plotting/plot_3d_map_to_surface_projection.py:254]:
```suggestion
# takes a nifti statistical map
```

File: examples/01_plotting/plot_3d_map_to_surface_projection.py [hunk 2]
```diff
@@ -88,18 +94,19 @@
...
-    bg_map=curv_right_sign,
+    bg_map=curv_sign,
     bg_on_data=True,
     engine=engine,  # Specify the plotting engine here
 )
 
-# Uncomment to display the figure
+# Uncomment the line below
+# to view the figure in browser.
 # figure.show()
```
Comment (bthirion) [inline examples/01_plotting/plot_3d_map_to_surface_projection.py:110]: In the previous stable doc, the plot was rendered...

**Review by bthirion — APPROVED** [review]
LGTM overall.

File: examples/01_plotting/plot_3d_map_to_surface_projection.py (hunk 2 above)
Comment (Remi-Gau) [reply examples/01_plotting/plot_3d_map_to_surface_projection.py:110]: - yeah we have this issue for a while on the doc build https://github.com/nilearn/nilearn/issues/4379

**Review by man-shu — APPROVED** [review]
LGTM!

**Pull request merged**